tracer/test/snapshots/CoverageRewriteTests.Rewritten.verified.txt

```diff
@@ -62,39 +63,50 @@ namespace CoverageRewriterAssembly
 				Console.WriteLine(MyProperty);
 				return;
 			}
-			scope.Report(2251838468915210uL, 2533330625560622uL);
+			counters[0]++;
+			counters[1]++;
 			Console.WriteLine("Main Method");
-			scope.Report(2814827077173275uL);
+			counters[2]++;
 			i = 0;
 			while (true)
 			{
-				scope.Report(2814874321813540uL);
+				counters[7]++;
 				if (i >= 100)
 				{
 					break;
 				}
-				scope.Report(3096280579112974uL, 3377772735758374uL);
+				counters[3]++;
+				counters[4]++;
 				Console.WriteLine(i);
-				scope.Report(3659230532665358uL, 2814912976519209uL);
+				counters[5]++;
+				counters[6]++;
```

Comment:
`counters[7]` is at the top, is that right? 🤔

Reply:
Yup, awesome right!? this is the  IL -> C# converter fault, the original is a `for` loop and the comparison is done after each loop that's why the debugging (sequencePoint) offset has this value. And when we try to get the C# code from that rewritten IL, the tool is unable to detect the `for` loop anymore.